.github/workflows/tests.yaml

```
@@ -189,8 +214,17 @@ jobs:
         if: steps.detect_if_should_run.outputs.result == 'true'
         run: make test.e2e
         env:
+          TEST_RUN: ${{ matrix.test }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           GOOGLE_APPLICATION_CREDENTIALS: ${{ secrets.GOOGLE_APPLICATION_CREDENTIALS }}
           GOOGLE_PROJECT: ${{ secrets.GOOGLE_PROJECT }}
           GOOGLE_LOCATION: ${{ secrets.GOOGLE_LOCATION }}
           KONG_LICENSE_DATA: ${{ steps.license.outputs.license }}
+
+  e2e-tests-passed:
+    needs: e2e-tests
+    if: ${{ always() }}
+    runs-on: ubuntu-latest
+    steps:
+    - name: e2e tests pased
+      run: echo all e2e tests passed
```

```suggestion
      run: echo all e2e tests passed

```
add missing new line at the end of file

---

Thanks!